src/sentry/api/endpoints/organization_metrics.py

```diff
@@ -178,6 +179,17 @@ class OrganizationMetricsDataEndpoint(OrganizationEndpoint):
     Based on `OrganizationSessionsEndpoint`.
     """
 
+    # still 40 req/s but allows for bursts of 200 up to req/s for dashboard loading
+    default_rate_limit = RateLimit(200, 5)
```

Comment:
nit: no need to define the variable here, I feel like duplicating it with shared constants would be better. Since it's not a variable defined to be accessed by the instance methods.